docs/tutorials/how-to-getting-started-with-estimator.ipynb

```diff
@@ -31,16 +31,25 @@
     "\n",
     "### Specify program inputs\n",
     "\n",
-    "The Estimator takes in the following arguments:\n",
+    "The Estimator.run() method takes in the following arguments:\n",
     "\n",
     "- **circuits**: a list of (parameterized) circuits that you want to investigate.\n",
     "- **observables**: a list of observables to measure the expectation values.\n",
-    "- **parameters**: a list of parameters for the parameterized circuits. It should be omitted if the circuits provided are not parameterized.\n",
+    "- **parameter_values**: an optional list of concrete parameters to be bound.\n",
     "- **skip_transpilation**: circuit transpilation is skipped if set to `True`. Default value is `False`.\n",
     "- **service**: the `QiskitRuntimeService` instance to run the program on. If not specified, the default saved account for `QiskitRuntimeService` is initialized.\n",
```

Comment:
`run()` doesn't take `service` or `skip_transpilation`. I think the tutorials will need a bit more info, since the steps are now 1) start a session 2) create an `Estimator` instance with the desired settings 3) submit a request with inputs and 4) get results back